docs/reference/changelog-r2021.md

```diff
@@ -17,6 +17,7 @@ Released on December XXth, 2020.
     - **The [InertialUnit](inertialunit.md)`.lookupTable` field was deprecated and the related `wb_inertial_unit_get_lookup_table_size` and `wb_inertial_unit_get_lookup_table` functions were removed ([#2424](https://github.com/cyberbotics/webots/pull/2424)).**
     - Added a `PlasticFruitBox` and a `MetalStorageBox` PROTO object ([#2427](https://github.com/cyberbotics/webots/pull/2427)).
     - Added a `data_type` parameter to a Python method [`getPointCloud`](lidar.md#wb_lidar_get_point_cloud) that retrieves a `bytearray` representation of points a few times faster than the previous version.
+    - Added a [`getDevice`](robot.md#wb_robot_get_device) method to the Python Robot API to retrieve devices in the same way as in C ([#2510](https://github.com/cyberbotics/webots/pull/2510)).
```

Comment:
In the documentation you also wrote that the `getAccelerometer`, etc. methods are deprecated.
This should also be mentioned in the ChangeLog.

Then a deprecation warning should also be printed in the Webots console when trying to use the old specific device methods.